website/docs/r/private_ip_mode.html.markdown

```diff
@@ -0,0 +1,46 @@
+---
+layout: "mongodbatlas"
+page_title: "MongoDB Atlas: private_ip_mode"
+sidebar_current: "docs-mongodbatlas-resource-private-ip-mode"
+description: |-
+    Provides a Private IP Mode resource.
+---
+
+# mongodbatlas_private_ip_mode
+
+`mongodbatlas_private_ip_mode` provides a Private IP Mode resource. This allows project to enable Connect via Peering Only mode.
```

Comment:
Modify:

provides a Private IP Mode resource. This allows one to enable/disable Connect via Peering Only mode for a MongoDB Atlas Project.